code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -93,6 +92,12 @@
 	var/input		//Thing that goes in
 	var/output		//Thing that comes out
 
+/datum/deepfryer_special/proc/validate(var/obj/item/I)
+	if(istype(I, input))
```

Comment:
Can do a simple return `istype(I, input)` here. The value of this will be a boolean value.